comictaggerlib/ctsettings/types.py

```diff
@@ -49,6 +49,13 @@ def user_log_dir(self) -> pathlib.Path:
             return path
         return pathlib.Path(super().user_log_dir)
 
+    @property
+    def user_plugin_dir(self) -> pathlib.Path:
+        if self.path:
+            path = self.path / "plugins"
+            return path
+        return pathlib.Path(super().user_config_dir)
```

Comment:
Is the `present working directory` expected to always be the root of comictagger? It may have been my scuffed setup but I had `comictaggerlib` as my `pwd` and so it was looking for `./comictagger/comictaggerlib/temp/plugins`